packages/terra-abstract-modal/tests/wdio/abstract-modal-spec.js

```diff
@@ -1,151 +1,147 @@
-      browser.setViewportSize({ height: 1684, width: 1870 });
+    it('clicks to open modal', () => {
+      browser.click('button');
+      Terra.validates.element('open', { selector });
     });
 
-    after(() => browser.setViewportSize(Terra.viewports('medium')[0]));
-
-    Terra.it.validatesElement({ selector: 'div[role="dialog"]' });
-  });
-
-  describe('Custom Props', () => {
-    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-custom-props'));
-    it('checks for custom id prop', () => {
-      expect(browser.getAttribute('div[role="dialog"]', 'id')).to.equal('custom-props');
+    it('clicks outside to close modal', () => {
+      browser.moveToObject('[class*="ModalOverlay"]', 235, 5)
+        .leftClick();
+      Terra.validates.element('closed');
+      expect(browser.hasFocus('#modal-open-button')).to.be.true;
     });
-  });
 
-  describe('Disable Close On Esc', () => {
-    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-disable-close-on-esc'));
-
-    Terra.it.validatesElement('modal present', { selector: 'div[role="dialog"]' });
+    it('reopens the modal', () => {
+      browser.keys('Enter');
+    });
 
-    it('attempts to close modal with ESC', () => {
+    it('closes modal on ESC', () => {
       browser.keys('Escape');
+      expect(browser.hasFocus('#modal-open-button')).to.be.true;
     });
-
-    Terra.it.validatesElement('modal present', { selector: 'div[role="dialog"]' });
   });
 
-  describe('Within Modal Focus Handling', () => {
-    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-custom-props'));
+  describe('Fullscreen Modal', () => {
+    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/fullscreen-abstract-modal'));
 
-    it('focuses on the modal when opened', () => {
-      expect(browser.hasFocus('[aria-modal="true"][role="dialog"]')).to.be.equal(true);
+    it('clicks to open modal', () => {
+      browser.click('button');
+      Terra.validates.element('open', { selector });
     });
 
-    Terra.it.validatesElement('modal is focused on open', { selector: '#root' });
-
-    it('focuses on interactive elements within the modal', () => {
-      browser.keys(['Tab']);
-      expect(browser.hasFocus('#focus-button')).to.be.equal(true);
+    it('clicks outside to close modal', () => {
+      browser.moveToObject('[class*="ModalOverlay"]', 235, 5)
+        .leftClick();
+      Terra.validates.element('closed');
+      expect(browser.hasFocus('#modal-open-button')).to.be.true;
     });
 
-    Terra.it.validatesElement('modal button focused', { selector: '#root' });
-
-    it('does not focus on interactive content within the app when the modal is open - tab cycle forward', () => {
-      browser.keys(['Tab']);
-      expect(browser.hasFocus('#focus-button')).to.be.equal(false);
+    it('reopens the modal', () => {
+      browser.keys('enter');
```

Comment:
We should test that the modal opened so that in the next test we know that if the modal is not open it was the escape key that closed it.

Reply:
Updated here: https://github.com/cerner/terra-framework/pull/723/commits/ab686cd3944fc4ac7fd14e94b77e79a183debbcb